website/content/docs/configuration/server.mdx

```diff
@@ -300,6 +300,8 @@ server {
   default_scheduler_config {
```

Comment:
It's out of scope for this PR, but I'm not wild about having punted this section out to the API docs. There's no reason to send operators looking thru the API docs to setup their cluster:

> The structure matches the [Update Scheduler Config][update-scheduler-config] API
endpoint, which you should consult for canonical documentation. However, the
attributes names must be adapted to HCL syntax by using snake case
representations rather than camel case.